app/main.py

```diff
@@ -2,24 +2,82 @@
 
 
 class IntegerRange:
-    pass
+
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+        self.private_name = None
+
+    def __set_name__(self, owner: str, name: str) -> None:
+        self.private_name = f"_{name}"
+
+    def __get__(self, instance: str, owner: None = None) -> None:
```

Comment:
The `__get__` method should return the value of the attribute from the instance, not `None`. The return type should be `int` instead of `None`.